GNUmakefile

```diff
@@ -0,0 +1,50 @@
+# Makefile for the supdup server and client.
+
+SHELL = /bin/sh
+PREFIX ?= /usr/local
+
+OS_NAME = $(shell uname)
+
+CC ?= cc
+CFLAGS += $(shell pkg-config --cflags ncurses 2> /dev/null) -g -Wall
+LDFLAGS += -g
+
+# AIX needs linking libcurses *after* libncurses
+ifeq ($(OS_NAME), AIX)
+ EXTRA_LDFLAGS = -lcurses
+endif
+
+# Mac OSX
+ifeq ($(OS_NAME), Darwin)
+ LDFLAGS = -L/opt/local/lib
```

Comment:
Doesn't this assume Macports?  Does it still work if one uses Homebrew?

Reply:
That came from the original Makefile... I don't have MacPorts installed and it worked fine for me in testing.